README.md

```diff
@@ -65,6 +67,13 @@ The explicit cache key to ferry to the
 `restore-keys` are not supported, because partial cache restoration leads to a
 ["snowball" effect](https://glebbahmutov.com/blog/do-not-let-npm-cache-snowball/).
 
+### Optional
+
+#### `read-only`
+
+Use this option to enable loading the docker-cache upon cache hit and disable
+saving the docker-cache upon cache miss.
```

Comment:
Multiple instances: this can be read to imply that this input has a bearing on the load step. It's generally advisable in the case of Boolean arguments to indicate which setting has which behavior and what the default is. See [the documentation for pre-commit-action's `bump` input](https://github.com/ScribeMD/pre-commit-action#bump) for an example.